### PR TITLE
Mappings for all indices

### PR DESCRIPTION
**Pull request by miiu96**
- Added mappings for all indices in order to optimize the size of the Elasticsearch cluster

### CODE REVIEW

File: templates/noKibana/transactions.go [hunk 1]
```diff
@@ -17,21 +17,103 @@ var Transactions = Object{
...
+			"fee": Object{
+				"index": "false",
+				"type":  "keyword",
+			},
+			"function": Object{
+				"type": "keyword",
+			},
+			"gasLimit": Object{
+				"index": "false",
+				"type":  "double",
```
Comment (bogdan-rosianu) [inline templates/noKibana/transactions.go:37]: maybe use another type for gas price / limit / used? for the entire PR

Comment (miiu96) [reply templates/noKibana/transactions.go]: done.

**Review by bogdan-rosianu — DISMISSED** [review]

File: templates/withKibana/operations.go [hunk 2]
```diff
@@ -20,13 +20,136 @@ var Operations = Object{
...
+			"originalTxHash": Object{
+				"type": "keyword",
+			},
+			"prevTxHash": Object{
+				"type": "keyword",
+			},
+			"receiver": Object{
+				"type": "keyword",
+			},
+			"receiverShard": Object{
```
Comment (mariusmihaic) [inline templates/withKibana/operations.go:96]: long?

File: templates/noKibana/blocks.go [hunk 3]
```diff
@@ -19,27 +19,231 @@ var Blocks = Object{
...
+				"type": "keyword",
+			},
+			"notarizedTxsCount": Object{
+				"index": "false",
+				"type":  "long",
+			},
+			"prevHash": Object{
+				"type": "keyword",
+			},
+			"proposer": Object{
```
Comment (mariusmihaic) [inline templates/noKibana/blocks.go:181]: Is it ok to be long here?

**Review by mariusmihaic — COMMENTED** [review]
In the future, please split lsuch large PR into smaller ones

File: templates/noKibana/operations.go [hunk 4]
```diff
@@ -20,13 +20,136 @@ var Operations = Object{
...
+				"type": "keyword",
+			},
+			"prevTxHash": Object{
+				"type": "keyword",
+			},
+			"receiver": Object{
+				"type": "keyword",
+			},
+			"receiverShard": Object{
+				"type": "keyword",
```
[the remark this answers is not in the document]
Comment (miiu96) [reply templates/noKibana/operations.go:97]: fixed

File: templates/noKibana/blocks.go (hunk 3 above)
Comment (miiu96) [reply templates/noKibana/blocks.go:181]: yes, because in this case propers in an index.

File: templates/withKibana/operations.go (hunk 2 above)
Comment (miiu96) [reply templates/withKibana/operations.go:96]: fixed

**Review by mariusmihaic — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by mariusmihaic — APPROVED** [review]

**Pull request merged**